### PR TITLE
build taipy frontend package simply

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
rename TAIPY_GUI_DIR to TAIPY_DIR

### CODE REVIEW

File: doc/gui/extension/example_library/front-end/scripts/install.js [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
+            }
+        )
+            .toString()
+            .trim();
+        return pipGuiDir.substring(9).trim();
+    } catch (e) {
+        log && console.info("taipy-gui pip package is not installed.");
+        const base = existsSync("package.json") ? `..${sep}..` : existsSync("frontend") ? "." : sep;
+        if (existsSync(resolve(base, "taipy", "gui", "webapp", "package.json"))) {
+            log && console.info(`Looking for taipy-gui npm package in ${resolve(base, "taipy", "gui", "webapp")}`);
```
Comment (FabienLelaquais) [inline doc/gui/extension/example_library/front-end/scripts/install.js]:
```suggestion
            log && console.info(`Found npm package for taipy-gui in ${resolve(base, "taipy", "gui", "webapp")}`);
```

File: doc/gui/extension/example_library/front-end/scripts/install.js [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
+            return base;
+        } else {
+            log && console.warn(`taipy-gui npm package should be build locally in ${resolve(base, "taipy", "gui", "webapp")} first.`);
```
Comment (FabienLelaquais) [inline doc/gui/extension/example_library/front-end/scripts/install.js]:
```suggestion
            log && console.warn(`taipy-gui npm package should be buit locally in ${resolve(base, "taipy", "gui", "webapp")} first.`);
```

**Review by FabienLelaquais — DISMISSED** [review]
Some 🔮pieces here and there, but hey...

File: doc/gui/extension/example_library/front-end/scripts/install.js (hunk 2 above)
Comment (FredLL-Avaiga) [reply doc/gui/extension/example_library/front-end/scripts/install.js]: buit or built 😶‍🌫️

**Review by FabienLelaquais — APPROVED** [review]
👌

**Review by dinhlongviolin1 — APPROVED** [review]
Lgtm

**Pull request merged**